src/components/Events.tsx

```diff
@@ -0,0 +1,79 @@
+import Link from 'next/link'
+import React from 'react'
+import type {EventData} from '../types/index'
+import DisplayRC from './DisplayRC'
+
+const events: EventData[] = [
+    {
+        "id": 1,
+        "title": "Physical Meetups",
+        "description": "Physical meetups where we share, learn and network.",
+        "image_url": "https://bit.ly/placeholder-img",
+        "target": "https://kommunity.com/reactjs-developer-community-kenya-reactdevske/events"
+      },
+      {
+        "id": 2,
+        "title": "Weekly Online Standups",
+        "description": "Weekly open calls in the community’s Telegram group discussing industry-related topics.",
+        "image_url": "https://bit.ly/placeholder-img",
+        "target": "https://t.me/joinchat/eNHADkJLCaMzZTM8"
+      },
+      {
+        "id": 3,
+        "title": "Community Power Sessions",
+        "description": "Community members meet and share technical concepts with other members of the community.",
+        "image_url": "https://bit.ly/placeholder-img",
+        "target": "https://www.youtube.com/channel/UC9_eVcPBk4T-DcZLHpQfy4w/videos"
+      },
+      {
+        "id": 4,
+        "title": "Joint Meetups",
+        "description": "Meetups hosted in collaboration with other tech communities.",
+        "image_url": "https://bit.ly/placeholder-img",
+        "target": "https://kommunity.com/reactjs-developer-community-kenya-reactdevske/events"
+      },
+      {
+        "id": 5,
+        "title": "Monthly Online Standups",
+        "description": "Monthly open calls on Google Meet  bringing together people from different communities.",
+        "image_url": "https://bit.ly/placeholder-img",
+        "target": "https://kommunity.com/reactjs-developer-community-kenya-reactdevske/events"
+      },
+      {
+        "id": 6,
+        "title": "Community Challenges",
+        "description": "We organize open source challenges to keep the community engaged.",
+        "image_url": "https://bit.ly/placeholder-img",
```

Comment:
Since this is the same image, also use `image_url: '/images/physical_meetups.jpg'`.